README.md

```diff
@@ -77,12 +77,77 @@ Cleans out/ folder if it exists else it's created, then it copies and vulcanizes
 grunt build
 ```
 
-#### Run non vulcanized version of TestRTC using (Google App Engine SDK for Python)(https://cloud.google.com/appengine/downloads). This is useful while developing. ####
+#### Run non vulcanized version of TestRTC using [Google App Engine SDK for Python](https://cloud.google.com/appengine/downloads). This is useful while developing. ####
```

Comment:
It starts to be too much text for the README... most of it not that relevant unless you are already quite into developing testrtc. I think we should move this to another file. Maybe more like a proper developers guide.

E.g. In the README.md I would expect a section about testing or adding new tests to be about how to add new testrtc tests and how to run only a specific test by setting test_filter=... etc... not about writing tests for the tests.